LukeMason33/Rotten-Tomatillos
# Feature responsiveness

### Is this a fix or a feature?

Feature.

### What is the change/fix?

This PR adds media queries for tablets, large phones, and regular/small phones. Files added to a new folder `/media-queries` in the `/sass` folder.

### Where should the reviewer start?

Review styling changes applied to the header, movie cards, and single movie page in each of the three new files.

### How should this be tested?

In browser, view mobile and cycle through different devices to ensure that styling is being applied correctly.
